Content.Server/Disease/DiseaseZombieSystem.cs

```diff
@@ -0,0 +1,188 @@
+        {
+            if (!component.ApplyZombieTraits || !HasComp<MobStateComponent>(uid))
+                return;
+
+            if (HasComp<DiseaseCarrierComponent>(uid))
+                EntityManager.RemoveComponent<DiseaseCarrierComponent>(uid);
+            if (HasComp<RespiratorComponent>(uid))
+                EntityManager.RemoveComponent<RespiratorComponent>(uid);
+            if (HasComp<BarotraumaComponent>(uid))
+                EntityManager.RemoveComponent<BarotraumaComponent>(uid);
+            if (HasComp<HungerComponent>(uid))
+                EntityManager.RemoveComponent<HungerComponent>(uid);
+            if (HasComp<ThirstComponent>(uid))
+                EntityManager.RemoveComponent<ThirstComponent>(uid);
+
+            EntityManager.EnsureComponent<BloodstreamComponent>(uid, out var bloodstream); //zoms need bloodstream anyway for healing
+            _bloodstream.SetBloodLossThreshold(uid, 0f, bloodstream);
+            _bloodstream.TryModifyBleedAmount(uid, -bloodstream.BleedAmount, bloodstream);
+            _movementSpeedModifier.RefreshMovementSpeedModifiers(uid);
+            EntityManager.EnsureComponent<ReplacementAccentComponent>(uid).Accent = "zombie";
+
+            if (TryComp<DamageableComponent>(uid, out var comp))
+            {
+                _damageable.SetDamageModifierSetId(uid, "Zombie", comp);
+                _damageable.SetAllDamage(comp, 0);
+            }
+
+            if(TryComp<HumanoidAppearanceComponent>(uid, out var spritecomp))
+            {
+                //zombie hex color is #aeb87bff
+                var color = new Color(0.70f, 0.72f, 0.48f, 1);
+
+                var oldapp = spritecomp.Appearance;
+                var newapp = oldapp.WithSkinColor(color);
+                _sharedHumanoidAppearance.UpdateAppearance(uid, newapp);
+				
+				_sharedHumanoidAppearance.ForceAppearanceUpdate(uid);
+            }
+
+            if(TryComp<HandsComponent>(uid, out var handcomp))
+            {
+                foreach (var hand in handcomp.Hands)
+                {
+                    _sharedHands.TrySetActiveHand(uid, hand.Key);
+                    _sharedHands.TryDrop(uid);
+
+                    var pos = EntityManager.GetComponent<TransformComponent>(uid).Coordinates;
+                    var virtualItem = EntityManager.SpawnEntity("ZombieClaw", pos);
+                    _sharedHands.DoPickup(uid, hand.Value, virtualItem);
+                }
+            }
+
+            if(TryComp<ContainerManagerComponent>(uid,out var cmcomp))
+            {
+                foreach (var container in cmcomp.Containers)
+                {
+                    if (container.Value.ID == "gloves")
+                    {
+                        foreach (var entity in container.Value.ContainedEntities)
+                        {
+                            container.Value.Remove(entity);
+                        }
+                    }
+                }
+            }
```

Comment:
sussy as all hell, what is this trying to do? remove gloves? go through inventory system for this instead of checking for a hardcoded container ID